openmc_plotter/plotmodel.py

```diff
@@ -116,10 +123,39 @@ def __init__(self):
         # reset random number seed for consistent
         # coloring when reloading a model
         reset_seed()
-
         self.previousViews = []
         self.subsequentViews = []
-        self.defaultView = self.getDefaultView()
+
+        if use_settings_pkl and os.path.isfile('plot_settings.pkl'):
+            with open('plot_settings.pkl', 'rb') as file:
+                data = pickle.load(file)
```

Comment:
I can make this a `try`/`except` block. Though I am confused why it would crash at this line specifically. Both are pickle files and should have no issue just being loaded. I would expect it to fail on the next lines.

Reply:
Ah, I see how it's failing with the attributes not existing. Definitely fixable w/ try/except